src/clients/apiClient.ts

```diff
@@ -105,7 +109,7 @@ export async function createSighting(newSighting: NewSighting): Promise<Response
 }
 
 export async function createNewUser(newUser: NewUser): Promise<Response> {
-  const response = await fetch(`${backendUrl}/users/create`, {
+  const response = await fetch(`${backendUrl}/likes/create`, {
```

Comment:
Oh I think this might have been a mistake?

Reply:
Fixed